test/amqp-integration-test.py

```diff
@@ -222,14 +223,32 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--no-node', dest="run_node", action="store_false",
```

Comment:
I think we wouldn't be doing any string joins in bash. In boulder/test.sh we would have:

`./test/integration-test.py`

And in letsencrypt/test.sh we would have:

`./test/integration-test.py --tests=python`

The inspiration for this comment is that, in test.sh, we started with the approach currently illustrated in this PR, having variables SKIP_UNIT_TESTS and SKIP_INTEGRATION_TESTS. It got cumbersome very quickly when we added RUN_MAKE.

```
matrix:
  - SKIP_INTEGRATION_TESTS=1
  - SKIP_UNIT_TESTS=1
  - RUN_MAKE=1 SKIP_UNIT_TESTS=1 SKIP_INTEGRATION_TESTS=1
```

Changing it to be a list of things to include (defaulting to "include all") simplified that a lot. Perhaps I'm fighting yesterday's battles! And I'm totally open to being convinced your way is better. Want to chat about it when you come in?

Another possibility that might satisfy both of us: what if we keep these as booleans, but change their sense, so they are `--node` and `--python`?